apps/exports/tests/test_runner.py

```diff
@@ -31,3 +32,29 @@ def test_custom_server_url(self):
         command = _compile_export_command(export_config, project, force=False)
 
         self.assertIn(server.url, command)
+
+
+class RunnerArgumentsDbTest(TestCase):
+
+    def test_default_server_url(self):
+        server = CommCareServer.objects.get()
+        project = CommCareProject(server=server, domain="foo")
+        account = CommCareAccount(server=server, username="foo", api_key="P@ssWord")
+        database = ExportDatabase(
+            name="Test DB", connection_string="postgresql://foo:bar@123.4.0.0/test"
+        )
+        config_file = TemporaryUploadedFile(
+            name="config_file", content_type="application/xml", size=100, charset="utf-8"
+        )
+        export_config = ExportConfig(
+            name="Test Config",
+            project=project,
+            account=account,
+            database=database,
+            config_file=config_file,
+            extra_args="",
+        )
+
+        command = _compile_export_command(export_config, project, force=False)
+
+        self.assertIn('https://www.commcarehq.org', command)
```

Comment:
Since the setting is modifiable, and we don't have a concept of "test settings", I'd use the constant here. 

```suggestion
        self.assertIn(settings.COMMCARE_DEFAULT_SERVER, command)
```

Reply:
This modification wouldn't be quite the same thing. Specifically when `COMMCARE_DEFAULT_SERVER` had a trailing slash, I wanted to test that we would still fix it. Which I think could also easily be rewritten to use your non-db test and an example.com url which is probably better.

The reason I wrote it this way was I wanted the test to explicitly be "does the default configuration result in a `Server` object that points at our valid production instance?". Do you think it'd be better to just unit test the slash removal and call it day?